docs/specification.md

```diff
@@ -1,150 +1,217 @@
 ---
-title: Seqcol specification version 0.0.1
+title: Seqcol specification version 0.1.0
 ---
 
 <!-- Table of contents: 
 * The generated Toc will be an unordered list
 {:toc} -->
 
-<h1>Seqcol specification version 0.0.1</h1>
+<h1>Seqcol specification version 0.1.0</h1>
 
 Table of contents:
 
 [TOC]
 
-NOTICE! THIS IS A DRAFT SPECIFICATION FOR FEEDBACK. NOTHING IN HERE IS OFFICIAL OR FINAL.
+## Specification version
+
+This specification is in **DRAFT** form. This is **NOT YET AN APPROVED GA4GH specification**.
 
 ## Introduction
 
-Reference sequences are fundamental to genomic analysis. The [refget protocol](http://samtools.github.io/hts-specs/refget.html) is a GA4GH standard that provides a way to access individual sequences using unique identifiers derived from sequences themselves. However, many applications require us to operate on *collections* of sequences. The *Sequence Collections*  (seqcol) protocol extends the refget protocol to collections of sequences. The seqcol protocol can be used to create unique identifiers derived from sequences collections themselves. Seqcol identifiers use refget identifiers under the hood.
+Reference sequences are fundamental to genomic analysis. To make analysis of reference sequences reproducible and efficient, we require tools that can identify, store, retrieve, and compare reference sequences. One such tool is the recent GA4GH standard [refget](http://samtools.github.io/hts-specs/refget.html), which provides a way to 1. compute deterministic sequence identifiers from sequences themselves, and 2) retrieve individual sequences using these unique identifiers derived. Refget thus provides a way to identify and retrieve sequences; however, many applications require working with *collections* of sequences. The *Sequence Collections* (seqcol) protocol is a pending GA4GH standard that works with collections of sequences. The seqcol protocol can be used to create unique identifiers derived from multiple sequences. Seqcol uses refget identifiers for individual sequences, adding new functionality to handle the complexity of collections, as well as attributes of sequences, such as names, lengths, or topologies. 
+
+The goal of the Seqcol project is **to standardize unique identifiers for collections of sequences**. It can be used to identify genomes, transcriptomes, or proteomes -- anything that can be represented as a collection of sequences. Seqcol uses a hash algorithm to generate a digest of the underlying collection of sequences. These unique identifiers are defined by an algorithm, rather than an accession authority, and are thus de-centralized and therefore usable for many purposes, including private or new sequence collections, cases without connection to a central database, or validation of sequence collection content and provenance.
```

Comment:
I'm not sure I would emphasise uniqueness here: seqcol identifiers are not primarily unique. For me the main goal is to create identifiers derived from the collection itself.